app/assets/stylesheets/datatable/table.scss

```diff
@@ -0,0 +1,12 @@
+table {
+  &.table-bordered {
+    border-right-width: 0;
+    &.dataTable {
+      th, td {
+        &:last-child {
+          border-right-width: 1px;
+        }
+      }
+    }
+  }
+}
```

Comment:
@Bunhok  Can you add an empty new line here?

Reply:
Yes, I can.